packages/adblocker-benchmarks/blockers/tsurlfilter.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2017-present Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const {
+  Engine,
+  Request,
+  RequestType,
+  RuleStorage,
+  StringRuleList,
+  setConfiguration,
+} = require('@adguard/tsurlfilter/dist/bundle.min.cjs');
+
+const requestTypes = new Map([
+  [ 'sub_frame', RequestType.Subdocument ],
+  [ 'script', RequestType.Script ],
+  [ 'stylesheet', RequestType.Stylesheet ],
+  [ 'image', RequestType.Image ],
+  [ 'xmlhttprequest', RequestType.XmlHttpRequest ],
+  [ 'media', RequestType.Media ],
+  [ 'font', RequestType.Font ],
+  [ 'other', RequestType.Other ],
+]);
+
+module.exports = class TSUrlFilter {
+  static async initialize() {
+    setConfiguration({
+      engine: 'extension',
+      version: '1.0.0',
+      verbose: true,
+    });
+  }
+
+  static async parse(rawLists) {
+    const list = new StringRuleList(1, rawLists, true, true, true);
```

Comment:
@ameshkov I'm passing `true` for all three options. I'm not sure if it makes a difference. In any case, all the filters we are passing here are URL filters (a subset of EasyList). Let me know if you see anything wrong with this.

Reply:
No worries, it's still a work in progress at this point, we're only entering the performance tuning stage.

Reply:
Yes, that's okay and it indeed does not make any difference.